WordPress/gutenberg
# Making size prop work for icon components using dash icon strings

## What?
Making size prop work for icon components using dash icon strings.


## Why?
Because currently is not working as the docs state: https://developer.wordpress.org/block-editor/reference-guides/components/icon/#size

## How?
Passing the prop to the child component and rendering the right styles/

## Testing Instructions

1. Import the icon component from `@wordpress/components` as:
`import { Icon }  from @wordpress/components;`
2. Render a dashicon like this:
`<Icon icon="plus-alt" size={10} />`
3. Check that the icon width and height are is10px.
